scripts/docs/json_builder.py

```diff
@@ -71,45 +93,58 @@ def signature_data(sig: inspect.Signature) -> dict:
     if return_annotation == inspect.Parameter.empty:
         return_annotation = None
 
-    return {
-        "params": params_data,
-        "return_type": str(return_annotation)
-    }
+    return {"params": params_data, "return_type": str(return_annotation)}
 
 
 def function_data(func: FunctionType) -> dict:
+    """
+    Parse the function object and return information about the function in a formatted dict
+    :param func: Function object to be analysed
+    :return:
+    """
     try:
         docstr_data = docstring_data(inspect.getdoc(func))
     except ValueError:
-        LOG.write(str(inspect.getmodule(func)) + ":" + str(func))  # logging invalid docstrings
-        LOG.write("\n")
+        print(f"WARN: Invalid docstring in func {str(inspect.getmodule(func))}: {str(func)}")
```

Comment:
Should not be necessary to do str(func) I am not sure if it is not necessary for str(inspect.getmodule(func)), please check it.